ilc/server/tailor/error-handler.js

```diff
@@ -37,15 +37,18 @@ function handleFragmentWarn(req, fragmentAttrs, err) {
     noticeError(new errors.FragmentWarn(errOpts));
 }
 
-/**
- * Setup error handlers for Tailor
- * @param {Tailor} tailor
- */
-module.exports = function setup(tailor) {
+const preSetupErrorHandlers = (errors, noticeError, errorHandler) => (tailor) => {
```

Comment:
Pls revert all the `function in function` things you've added here. Same thing can be done in a much simpler way.
```
module.exports = function setup(tailor, errorHandler, noticeError) {
```

And in `tailor/factory.js` you will have:
```

const appErrorHandler = require('../errorHandler');
const noticeError = require('../errorHandler/noticeError');
const errorHandlerSetup = require('./error-handler');

errorHandlerSetup(tailor, appErrorHandler, noticeError);
```

Reply:
I changed setup function, reverted error handlers, and moved them into setup function to use dependencies from there.